actionners/kubernetes/drain/drain.go

```diff
@@ -158,7 +157,7 @@ func (a Actionner) RunWithClient(client k8s.DrainClient, event *events.Event, ac
 	nodeName := node.GetName()
 	objects["node"] = nodeName
 
-	pods, err := client.ListPods(context.Background(), metav1.ListOptions{
+	pods, err := client.ListPods(metav1.ListOptions{
```

Comment:
I'd rather have todo here so we can at least remember to change it in the future instead of removing

Reply:
as you wish